nplh.go

```diff
@@ -24,17 +24,22 @@ func done(msg string) {
 	fmt.Println(green("✔") + " " + gray(msg))
 }
 
-func resolvePath(path string) (newpath string) {
-	usr, _ := user.Current()
+func resolvePath(path string) (string, error) {
+	usr, err := user.Current()
+	if err != nil {
+		return "", err
+	}
+
 	dir := usr.HomeDir
 
 	if strings.HasPrefix(path, "~/") {
 		path = filepath.Join(dir, path[2:])
 	}
-	return path
+
+	return path, nil
 }
 
-func fileExists(path string) (exists bool) {
```

Comment:
Why did you remove this?

Reply:
I didn't... I removed the named return value exists since you aren't using it. I really just modified it.

Reply:
Why did you remove the named from the return value is what I am asking

Reply:
Isn't it useful for autocompletion?

Reply:
It's useful for documentation if you are returning something that is not obvious, but since you aren't using the automatically created named return value and it's obvious that `fileExists` returns whether the file exists, I think it should be removed.

Reply:
OK, fair